package.json

```diff
@@ -51,8 +52,11 @@
     "babel-root-import": "^4.1.8",
     "css-loader": "^0.27.3",
     "file-loader": "^0.10.1",
+    "font-awesome": "^4.7.0",
+    "font-awesome-webpack-2": "^1.0.0",
```

Comment:
Is this in both the dependencies and the peer dependencies for a reason?  Also, should `font-awesome` be a peer dependency as well?